development/sass/top-nav.scss

```diff
@@ -0,0 +1,72 @@
+.navbar {
+  background-color: #2d2d30;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  height: 50px;
+  letter-spacing: 2px;
+  line-height: 20px;
+  opacity: 0.9;
+  text-transform: uppercase;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+
+  button {
+    display: none;
+    padding: 3px 7px;
+    margin: 6px 5px;
+    background-color: #2d2d30;
+    border-radius: 4px;
+  }
+
+  button:focus {
+    outline: none;
+    background-color: #ddd;
+  }
+
+  button i {
+    color: #888;
+  }
+
+  ul {
+    padding: 15px;
+    margin: 0;
+  }
+
+  li {
+    display: inline-block;
+    float: right;
+    list-style-type: none;
+  }
+
+  li a, div a {
+    color: #d5d5d5;
+    font-size: 14px;
+    text-decoration: none;
+    padding: 15px;
+  }
+
+  li a:hover, div a:hover {
+    color: #fff;
+  }
+
+  .navbar-brand {
+    font-size: 18px;
+    float: left;
+  }
+
+  a:-webkit-any-link {
```

Comment:
I'd declare this closer to the top and just us `a {...` no need for the website thing.